documentation/Using-the-SDK/Creating-your-plugin.md

```diff
@@ -10,46 +10,24 @@ This document will cover:
 
 
 
-For simplicity, this section will assume you are using Visual Studio 2019. The instructions may be adapted for other editors / IDEs.
+For simplicity, this section will assume you are using Visual Studio. The instructions may be adapted for other editors / IDEs.
 
 <a name="reqs"></a>
 # Requirements
 
-1. [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+1. [Visual Studio](https://visualstudio.microsoft.com/downloads/)
 2. [.NET SDK that supports .NET Standard 2.0](https://dotnet.microsoft.com/download/visual-studio-sdks)
    * [See .NET Standard 2.0 support options](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
 
-## Visual Studio 2019
-
-You can install Visual Studio 2019 from here: [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/).
-Any edition is capable of creating an SDK plugin.
-
-1. Download the installer
-2. Execute the installer
-3. Make sure that the following features are selected:
-   * .NET Desktop Development (Make sure to select an appropriate .NET SDK for development option)
-4. Click "Install"
-5. You may have to restart your computer when installation finishes
-
-## .NET Standard 2.0
```

Comment:
I don't think we need it. .NET comes installed with VS, so most people don't need to do this step. Since we're already linking to the landing page for the requirements, I think it's better to have customers defer to those sites for installation help. Installation instructions change over time, so keeping our docs in sync with external programs is a bit much